### DISCUSSION TITLE
How can i set aliases for @EventType on dashboard?

### DISCUSSION THREAD

**Discussion by XaspMe** [datalust/seq-tickets#1306] [Help] [answered]
Hello!
I have this query in my dashboard, It displays several points on one chart.
How can i set aliases for @EventType? 
![Example](https://user-images.githubusercontent.com/48912208/129691749-162b44ff-6b61-48fc-8609-206776481a1b.png)
select count(*) as col0
from stream 
where 
@EventType = 0xCB325EAA or  @EventType = 0x0969c1fd or @EventType = 0x071d713a
group by @EventType, time(15m)
limit 10000

Il try 'SELECT * FROM (some nested query)', but get:
Syntax error (line 2, column 1): unexpected keyword `select`, expected `stream`.

**Comment by nblumhardt** [comment, marked as answer]
Hi! It's a bit clunky, but this should work :-)

```
select count(*) as col0
from stream
where @eventtype = 0xCB325EAA or @eventtype = 0x0969c1fd or @eventtype = 0x071d713a
group by
  if @eventtype = 0xCB325EAA then 'first' else if @eventtype = 0x0969c1fd then 'second' else 'third' as eventtype,
  time(15m)
limit 10000
```

Let me know if this helps,
Nick
